client/package.json

```diff
@@ -32,12 +32,28 @@
     "@babel/runtime": "^7.8.7",
     "@react-native-community/eslint-config": "^0.0.7",
     "babel-jest": "^25.1.0",
```

Comment:
I can't comment on package-lock.json but we also don't commit package-lock.json, package.json is fine here

Reply:
Also we don't have to commit yarn.lock

Reply:
package.-lock.json will protect our developer from dependency confilcts

Reply:
good to know. But we will use yarn, using npm cannot fix dependencies warning whenever a package goes bad, yarn updates our packages without having to use npm audit fix every time we encounter a vulnerability issue.

Reply:
please remove your package-lock.json file and use yarn install instead, I hesitated from switching to yarn but after using it, I think that it's better than npm, just for the vulnerability fixes.

Reply:
removed. at commit e83182a